lti_redirect/views.py

```diff
@@ -1,15 +1,23 @@
+import random, logging
+import string
 import requests
 import jwt
+import django.contrib.auth
+from django.conf import settings
 from django.shortcuts import redirect, render
 from lti_tool.views import LtiLaunchBaseView
+from django.contrib.auth.models import User
+
+logger = logging.getLogger(__name__)
 
```

Comment:
I understand the `view.py` is handling lot of things, But once POC is complete we can refactor the code.